README.md

```diff
@@ -36,7 +37,7 @@ appropriate one:
 Installation
 ------------
 
-    pip install git+https://github.com/hastexo/tutor-contrib-s3@v1.2.0
+    pip install git+https://github.com/hastexo/tutor-contrib-s3@v1.3.0
```

Comment:
Exclude this change, please. This goes into a separate commit created by `bump2version` when we cut the release.